contents/blog/the-posthog-array-1-37-0.mdx

```diff
@@ -0,0 +1,146 @@
+import DancingHog from '../images/blog/array/1-37-0-dancing-hog.gif'
+import ConfigurableColumns from '../images/blog/array/1-37-0-configurable-columns.png'
+import APICompression from '../images/blog/array/1-37-0-compression.png'
+
+Want to know more about what we're up to? [Subscribe to HogMail, our new newsletter](/newsletter), which we send out every two weeks!
+
+> Running a self-hosted instance? Check out our [Upgrading PostHog guide](/docs/self-host/configure/upgrading-posthog).
+
+## PostHog 1.37.0 release notes
+
+**Release highlights:**
+
+- New: [Cohorts 2.0](#new-cohorts-2.0)
+- New: [Event and property detail pages](#new-event-and-property-detail-pages)
+- New: [Dancing hedgehogs](#new-dancing-hedgehogs)
+- Improved: [Save event columns](#improved-save-event-columns)
+- Improved: [Faster response times](#improved-faster-response-times)
+
+### New: Cohorts 2.0
+
+Cohorts are an extremely powerful tool. They let you define a group of users to later base your analytics on. In this release, we completely revamped how you define cohorts, giving you flexible and powerful filtering capabilities.
+
+<img src={Cohorts2} alt="Cohorts 2.0" style={{ width: 720 }} />
+
+#### New condition types
+
+Previously, cohorts could only be defined using the following conditions:
+* Match users who have a certain property
+* Match users who have completed a certain event
+
+We've now added the following conditions:
+* **Did not complete event:** Find users who aren't doing what you expect. For example, "Give me users who visited the home page, but did not click on the 'Sign up' button.".
+* **Completed an event multiple times:** Find your most active users. For example, "Give me users who have 'Bought item' more than 3 times in the last 30 days".
+* **Completed a sequence of events:** Find users using your product in a very specific way. For example, "Give me users who added something to their cart and then entered a promo code within a day".
+* **Did not complete a sequence of events:** Find users who aren't using your product as you'd like. For example, "Give me users who added something to their cart, but didn't checkout within a day".
+* **Do not have the property:** Find more specific sets of users. For example, "Give me users outside of Europe".
+* **Completed an event for the first time:** Find the newest users of a feature. For example, "Give me users who bought an item for the first time in the last 7 days".
+* **Completed an event regularly:** Find your power users. For example, "Give me users who bought an item in 5 out of the last 7 weeks".
+* **Stopped doing an event:** Find your users that are at risk of churning. For example, "Give me users who haven't bought anything in the last 7 days, but had bought something in the 30 days prior".
+* **Started doing an event again:** Find your users that did not churn. For example, "Give me users who bought something, then didn't for 2 weeks, but have again in the last 7 days".
+
+#### AND/OR operators
```

Comment:
Maybe it could be mentioned here that AND/OR has been introduced to PostHog in 1.36.0